src/main/java/org/example/models/UserInterface.java

```diff
@@ -0,0 +1,181 @@
+        for (int i = 0; i < visits.size(); i++) {
+            if (docID == visits.get(i).getDoctorId()) {
+                System.out.println(visits.get(i).toString());
+            }
+        }
+    }
+
+
+    public void run(List<Patient> patients, List<Doctor> Mds, List<MedicalVisit> visits) {
+        mainMenu();
+        switch (getValue()) {
+            case 1: {
+                doctorPath(patients, Mds, visits);
+                break;
+            }
+            default: {
+                patientPath(patients, Mds, visits);
+            }
+
+
+        }
+
+    }
+
+    public void setVisit(List<Patient> patients, List<MedicalVisit> visits, int ID, Boolean isDoctor) {
+
+        System.out.println("Select a month");
+        int month = getValue();
+        System.out.println("Select a day");
+        int day = getValue();
+        System.out.println("Select a hour");
+        int hour = getValue();
+        System.out.println("Select a min");
+        int min = getValue();
+        if (isDoctor ==true) {
+            System.out.println("Enter patients id");
+            int sickOne = getValue();
+            MedicalVisit tempvisit = new MedicalVisit(visits.size() + 1, ID, sickOne, month, day, hour, min, VisitStatus.ACTIVE);
+            visits.add(tempvisit);
+        }
+        if (isDoctor== false){
+            System.out.println("Enter patients/Your id");
+            int sickOne = getValue();
+            System.out.println("Enter Your MD id");
+            int docID = getValue();
+            MedicalVisit tempvisit = new MedicalVisit(visits.size() + 1, docID, sickOne, month, day, hour, min, VisitStatus.ACTIVE);
+            visits.add(tempvisit);
+        }
+        System.out.println("done.");
+    }
+
+    private void doctorPath(List<Patient> patients, List<Doctor> Mds, List<MedicalVisit> visits) {
+        while (true) {
+            boolean isDoctor = true;
+            Doclist(Mds);
+            chooseUser();
+            int docID = getValue();
+            Visitlist(Mds, patients, visits, docID, isDoctor);
+            actionsMenudoctor();
+
+            switch (getValue()) {
+                case 1: {
+                    setVisit(patients, visits, docID, isDoctor);
+                    break;
+                }
+                case 2: {
+                    int pointer = 2;
+                    deleteVisit(patients, visits, docID, pointer);
+                    break;
+                }
+                case 3: {
+                    int pointer = 3;
+                    deleteVisit(patients, visits, docID, pointer);
+                    break;
+                }
+                default: {
+                    Visitlist(Mds, patients, visits, docID, isDoctor);
+                }
+
+            }
+
+        }
+    }
+    private void patientPath(List<Patient> patients, List<Doctor> Mds, List<MedicalVisit> visits) {
```

Comment:
do usunięcia niepotrzebny kod

Reply:
Naprawdę? przecież to jest część "logiki biznesowej"